cmake/package.cmake

```diff
@@ -0,0 +1,43 @@
+if(TARGET Cryptopp::static_library)
+    return()
+endif()
+
+set(
+    CRYPTOPP_INCLUDE_PATH
+    "${CMAKE_CURRENT_LIST_DIR}/include"
+)
+
+set(
+    CRYPTOPP_STATIC_LIBRARY_PATH
+    "${CMAKE_CURRENT_LIST_DIR}/lib"
+)
+
+add_library(
+    Cryptopp::static_library
+    STATIC
+    IMPORTED
+)
+
+if(MSVC)
+    set_target_properties(
+        Cryptopp::static_library
+        PROPERTIES
+        IMPORTED_LOCATION
+            "${CRYPTOPP_STATIC_LIBRARY_PATH}/libcryptopp.lib"
+        IMPORTED_LOCATION_DEBUG
+            "${CRYPTOPP_STATIC_LIBRARY_PATH}/libcryptoppd.lib"
```

Comment:
it looks like lib-file names are wrong

Reply:
You are right. The original lib name for windows is "cryptlib.lib". I corrected libs name.